src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < 3; i++) {
+            figureSupplier.getRandomFigure().toDraw();
+        }
+        for (int i = 0; i < 10; i++) {
+            figureSupplier.getDefaultFigure().toDraw();
+        }
```

Comment:
task says you should create and array and fill one half of it with random figures and one half with default figures